admin/page/ajax-edit-produk.php

```diff
@@ -0,0 +1,264 @@
+                    <option <?php if($kategori=="Grenade Launcher"){echo "selected";} ?> >Grenade Launcher</option>
+                    <option <?php if($kategori=="Mortir"){echo "selected";} ?> >Mortir</option>
+                    <option <?php if($kategori=="Pistols"){echo "selected";} ?> >Pistols</option>
+                    <option <?php if($kategori=="Shotguns"){echo "selected";} ?> >Shotguns</option>
+                    <option <?php if($kategori=="Submachine Guns"){echo "selected";} ?> >Submachine Guns</option>
+                </select>
+            </div>
+            <?php
+            add_spesifikasi();
+            add_deskripsi();
+            break;
+
+        case 'Munisi':
+            ?>
+            <div class="form-inline">
+                <label>Kategori</label>
+                <select class="form-control ml-auto" name="kategori" required>
+                    <option <?php if($kategori=="Small Caliber"){echo "selected";} ?> >Small Caliber</option>
+                    <option <?php if($kategori=="Big Caliber"){echo "selected";} ?> >Big Caliber</option>
+                    <option <?php if($kategori=="Special Ammunition"){echo "selected";} ?> >Special Ammunition</option>
+                </select>
+            </div>
+            <?php
+            add_spesifikasi();
+            add_deskripsi();
+            break;
+
+        case 'Kendaraan Khusus':
+            ?>
+            <div class="form-inline">
+                <label>Kategori</label>
+                <select class="form-control ml-auto" name="kategori" required>
+                    <option <?php if($kategori=="4 x 4 Komodo"){echo "selected";} ?> >4 x 4 Komodo</option>
+                    <option <?php if($kategori=="6 x 6 Anoa"){echo "selected";} ?> >6 x 6 Anoa</option>
+                </select>
+            </div>
+            <?php
+            add_spesifikasi();
+            add_deskripsi();
+            break;
+
+        case 'Alat Berat':
+            add_spesifikasi();
+            add_deskripsi();
+            break;
+
+        case 'Peralatan Industi dan Jasa':
+            ?>
+            <div class="form-inline">
+                <label>Kategori</label>
+                <select class="form-control ml-auto" name="kategori" required>
+                    <option <?php if($kategori=="Electric Machines"){echo "selected";} ?> >Electric Machines</option>
+                    <option <?php if($kategori=="Agricultural Equipments"){echo "selected";} ?> >Agricultural Equipments</option>
+                </select>
+            </div>
+            <?php
+            add_spesifikasi();
+            add_deskripsi();
+            break;
+        
+        case "Insfrastruktur Perhubungan":
+            ?>
+            <div class="form-inline">
+                <label>Kategori</label>
+                <select class="form-control ml-auto" name="kategori" required>
+                    <option <?php if($kategori=="Forging"){echo "selected";} ?> >Forging</option>
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)